### ISSUE TITLE
Switch to AEM Mocks 2.7.0 with Content Policy Support

### ISSUE THREAD

**Issue by stefanseifert** [adobe/aem-core-wcm-components#682]
we recently added content policy support to [AEM Mocks](https://wcm.io/testing/aem-mock/) - testable with version 2.6.1-SNAPSHOT. as you are using both AEM Mocks and (a homegrown support for) content policies in the unit tests for the core components you may want to give it a try to simplify your unit tests.

usage is really simple - if you want to provide a component with a content policy for testing, just add a mapping for its resource type with a code line like this:

```
// create a content policy with mapping for resource type
context.contentPolicyMapping("app1/componenty/component1",
    "prop1", "value1",
    "prop2", 123);
```

it is then available either using the ContentPolicy APIs, or when injecting a "currentStyle" in a sling model within the unit test.

release is planned in 1-2 weeks.

**Comment by andreeadracea** [comment]
(CQ-4276677)

**Comment by stefanseifert** [comment]
AEM Mocks 2.7.0 is now released.
https://wcm-io.atlassian.net/wiki/x/AQCLS

**Comment by richardhand** [comment]
Code is merged to [master](https://github.com/adobe/aem-core-wcm-components/commits/master) and will be available in the upcoming [2.6.0 Release](https://github.com/adobe/aem-core-wcm-components/milestone/16).
